File: examples/rss/list/go/listRssFeeds.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/rss"
	"os"
)

// List the saved RSS fields
func ListRssFeeds() {
	rssService := rss.NewRssService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := rssService.List(&rss.ListRequest{})
	fmt.Println(rsp, err)
}

File: examples/rss/list/node/listRssFeeds.js
const { RssService } = require("m3o/rss");

// List the saved RSS fields
async function listRssFeeds() {
  let rssService = new RssService(process.env.MICRO_API_TOKEN);
  let rsp = await rssService.list({});
  console.log(rsp);
}

listRssFeeds();
